openc3/lib/openc3/models/target_model.rb

```diff
@@ -442,11 +445,6 @@ def undeploy
       Store.del("#{@scope}__openc3tlm__#{@name}")
       Store.del("#{@scope}__openc3cmd__#{@name}")
 
-      # Note: these match the names of the services in deploy_microservices
-      %w(DECOM COMMANDLOG DECOMCMDLOG PACKETLOG DECOMLOG REDUCER CLEANUP).each do |type|
-        model = MicroserviceModel.get_model(name: "#{@scope}__#{type}__#{@name}", scope: @scope)
-        model.destroy if model
-      end
```

Comment:
I think this is unnecessary due to the code in plugin_model undeploy

Reply:
Not unnecessary from a standalone perspective.  Needs to be added back.

Reply:
I will restore.